### DISCUSSION TITLE
some problem with "doodson.cpp"

### DISCUSSION THREAD

**Discussion by MrSoul8715** [groops-devs/groops#229] [Q&A] [answered]
Hi, I've some problem with the "nodeCorr" method in "doodson.cpp" when read the souce code. Would do you tell me about the reference aticle about IHO correction method and Schureman correction method. Thanks for reading.

**Comment by tmayerguerr** [comment, marked as answer]
Hi, 
can you specify your problem? Is something being calculated incorrectly or are you having difficulty understanding the code? The "nodeCorr" code is quite old and the programmer is no longer at our institute. I don't have any further information besides the comments in the code. We no longer use this code at all, but calculate the admittance with a matrix as described in our published ocean time models, see [Gravitational effect of ocean and atmospheric tides](https://ifg.tugraz.at/downloads/ocean-tides).
regards
Torsten

**Comment by MrSoul8715** [comment]
Thanks very much for your repley! I just don't find the reference material which contain those formula correponds to those code. I just want to make it more clearer. Thanks for reading.
